packages/orbit-tailwind-preset/src/presets/components/index.ts

```diff
@@ -0,0 +1,179 @@
+import type { Config } from "tailwindcss";
+import { defaultTokens } from "@kiwicom/orbit-design-tokens";
+import plugin from "tailwindcss/plugin";
+
+import orbitFoundationPreset from "../..";
+import {
+  kebabCase,
+  getComponentLevelToken,
+  ExportedComponentLevelTokens,
+} from "../foundation/helpers";
+
+const COLORS: Partial<ExportedComponentLevelTokens>[] = [
+  "button",
+  "buttonLink",
+  "socialButton",
+  "alert",
+  "icon",
+  "formElement",
+  "badge",
+  "tag",
+  "textLink",
+  "text",
+  "heading",
+];
+
+interface Options {
+  /** default: true e.g. does not include default normalize styles */
+  disablePreflight?: boolean;
+  /** default: [] */
+  content?: Config["content"];
+}
+
+const cfg = (options?: Options): Config => {
+  const { disablePreflight = true } = options || {};
+
+  return {
+    content: ["auto"],
+    presets: [orbitFoundationPreset],
+    corePlugins: {
+      preflight: disablePreflight ? false : undefined,
+    },
+    theme: {
+      extend: {
+        colors: {
+          ...COLORS.reduce((acc, name) => {
+            // eslint-disable-next-line no-param-reassign
+            acc = {
+              ...acc,
+              ...getComponentLevelToken(name, "background"),
+              ...getComponentLevelToken(name, "backgroundHover"),
+              ...getComponentLevelToken(name, "backgroundActive"),
+              ...getComponentLevelToken(name, "foreground"),
```

Comment:
While migrating the Heading component to Tailwind, I noticed that the foreground-inverted class was not being generated. Should it be added here? Or since only Heading (for now) has this rule we should just add it manually at the bottom of this object, as we do with other classes in other properties? 🤔

Reply:
I think for now we can use the paletteWhite color, as the token does not exists for that inside Orbit tokens package

Reply:
I'm not sure I understand. The token is defined [here](https://github.com/kiwicom/orbit/blob/feat/tailwind-config/packages/orbit-design-tokens/src/dictionary/definitions/component/heading/heading-foreground.json#L8). And if we have a token for it we should use it 💭

Reply:
Ah, right, did not check that as I assumed we do not have it inside definition. Yeah, will make sense actually to spread in the same way (maybe in future we can use it for other components in dark mode, for instance)